index.js

```diff
@@ -57,7 +57,7 @@ function paginate (filePath, collection, fileName, files, iteratee) {
     });
 }
 
-module.exports = function(opts) {
+module.exports = (opts) => {
```

Comment:
Mind elaborating what we gain from doing this?

Reply:
Just consistency. I missed it the other time.